### ISSUE TITLE
pyFFTW import fails with scipy1.4.0

### ISSUE THREAD

**Issue by nbara** [pyFFTW/pyFFTW#279]
The latest scipy release included a refactoring of the FFT module:
https://docs.scipy.org/doc/scipy-1.4.0/reference/fftpack.html#module-scipy.fftpack

This causes pyfftw to break on import. 

```python
import pyfftw
File "C:\hostedtoolcache\windows\Python\3.7.5\x64\lib\site-packages\pyfftw\__init__.py", line 43, in <module>
    from . import interfaces
  File "C:\hostedtoolcache\windows\Python\3.7.5\x64\lib\site-packages\pyfftw\interfaces\__init__.py", line 237, in <module>
    from . import scipy_fftpack
  File "C:\hostedtoolcache\windows\Python\3.7.5\x64\lib\site-packages\pyfftw\interfaces\scipy_fftpack.py", line 65, in <module>
    from scipy.fftpack import (dct, idct, dst, idst, diff, tilbert, itilbert,
ImportError: cannot import name '_fftpack' from 'scipy.fftpack' (C:\hostedtoolcache\windows\Python\3.7.5\x64\lib\site-packages\scipy\fftpack\__init__.py)
```

**Comment by hgomersall** [comment]
ach, thanks for that. Should be a fairly simple fix, but we don't want to break previous releases.

**Comment by hgomersall** [comment]
@grlee77 It seems `interfaces.__init__.py` always imports `scipy_fftpack`, is that correct?

**Comment by grlee77** [comment]
I will take a quick look. We have not made a release yet with the changes adding the new `scipy.fft` backend so I can do that if needed. I think the wheels repo was passing on Python 3.8 wheels now to, so we can also add Python 3.8 support at the same time.

**Comment by grlee77** [comment]
Okay, I recall now. We have actually already fixed this import of the non-public `_fftpack` in master. We just need to push out a new release. Does that sound good to you @hgomersall?

**Comment by grlee77** [comment]
@nbara: 
Until we make a new release (hopefully in the next few days) you will need to either stay with scipy 1.3.x or build pyFFTW from source.

**Comment by hgomersall** [comment]
Yeah, I misread the original issue. Am I right in asserting that `fftpack` still exists?

**Comment by grlee77** [comment]
Yes, the `scipy.fftpack` namespace will be retained long-term for purposes of backwards compatibility. The actual Fortran FFTPack code underlying it has been removed from SciPy as of 1.4.0, but the public functions still exist and just call `pocketfft` under the hood now instead.

**Comment by mhvk** [comment]
Found this because our docs are not building, and was very happy to see a fix was in the works!

One suggestion might be to guard against import problems of `interfaces` more generally, since ideally changes in numpy or scipy implementation should not determine whether or not the regular FFTW wrappers can be used. That said, presumably a change like the one here is quite rare!

**Comment by mrava87** [comment]
Hi, do you have an idea when the new release will be published so that pyFFTW can work with latest 1.4.x scipy?

**Comment by grlee77** [comment]
Hi @hgomersall, what is your availability this coming week? I think we are (nearly) ready to tag and release. I am happy to do it, but will need your help to review any PRs here and at pyFFTW-wheels since the repo settings currently do not allow me to merge without review.

**Comment by grlee77** [comment]
closed by #265 (included in the v0.12.0 release)
